OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -90,3 +99,64 @@ persistence:
   # volumeConfiguration:
   #   hostPath:
   #     path: /data/nxiq/
+
+# configYaml is the full text of the config.yml file that will be passed to IQ Server
+configYaml:
```

Comment:
The README.md needs to be updated to reference all of the new configuration labels, the new `configYaml` format and all of the new features you've added.